### PR TITLE
[FEATURE] Built Footer Section solve #59

### PR DESCRIPTION
**Pull request by RohanDarade**
image
image
image
image
image
image

### CODE REVIEW

File: src/components/Footer/index.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import Youtube from "../../../static/img/FooterSectionIcons/youtube.svg";
+
+function Footer() {
+  return (
+    <div className="bg-[#07001D]">
+      <div className="flex px-10 py-10 w-screen flex-col md:flex-row justify-center items-center lg:justify-between flex-wrap">
+        <div className="flex items-center">
+          <img src={Logo} alt="tcet-icon" className="h-12 w-12 inline" />
+          <div className="inline text-[#FCFCFD] font-semibold text-2xl mx-4">
+            TECT Open Source
```
Comment (riyavishwa981) [inline src/components/Footer/index.js:15]: Make it TCET

File: src/components/Footer/index.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+          </a>
+          <a
+            href="*"
+            className="hover:no-underline hover:text-[#FCFCFD] mx-4 my-3"
+          >
+            Contact
+          </a>
+        </div>
+      </div>
+      <div className="flex px-10 pb-12 w-screen flex-col md:flex-row justify-center items-center lg:justify-between flex-wrap">
```
Comment (riyavishwa981) [inline src/components/Footer/index.js:57]: If you see in the design there is line in between 
image

**Review by riyavishwa981 — CHANGES_REQUESTED** [review]

**Comment by riyavishwa981** [comment]
Rest everything is good

**Comment by JayeshVP24** [comment]
make these changes soon @RohanDarade

**Comment by JayeshVP24** [comment]
href links are all incorrect.
make the right changes
youtube https://www.youtube.com/@TCETMumbaiOfficial
find IG, twitter and linkedIn by yourself.

**Review by JayeshVP24 — APPROVED** [review]
LGTM

**Pull request merged**